packages/frontend/src/router.definition.ts

```diff
@@ -57,18 +76,14 @@ const routes: RouteDef[] = [{
 		path: '/avatar-decoration',
 		name: 'avatarDecoration',
 		component: page(() => import('@/pages/settings/avatar-decoration.vue')),
-	}, {
-		path: '/roles',
-		name: 'roles',
-		component: page(() => import('@/pages/settings/roles.vue')),
 	}, {
```

Comment:
同様に

このsuggestionとother.vueの`markerId`のsusggestionはペアです。

```suggestion
	}, {
		path: '/roles',
		redirect: '/settings/other#roles',
	}, {
```